### ISSUE TITLE
Segfaults in ContainersUnitTest_OpenMP and ContainersPerformanceTest_OpenMP unit tests

### ISSUE THREAD

**Issue by rbberger** [kokkos/kokkos#7285]
While trying to package Kokkos 4.4.00 for Fedora, I'm noticing multiple OpenMP unit test segfaults. Build logs can be found here: 

https://koji.fedoraproject.org/koji/taskinfo?taskID=122857695

**Comment by dalg24** [comment]
Sounds a whole lot like #7283 
The compile lines have `-fvisibility=hidden -fvisibility-inlines-hidden` and the segfaults seem to be happening with `ScatterView`.

**Comment by dalg24** [comment]
We confirmed it was the same symbol visibility issue and are working on a 4.4.1 patch release.
In the meantime you should consider doing
```diff
diff --git a/cmake/kokkos_tribits.cmake b/cmake/kokkos_tribits.cmake
index 6da543a2c..1ea842e77 100644
--- a/cmake/kokkos_tribits.cmake
+++ b/cmake/kokkos_tribits.cmake
@@ -160,12 +160,6 @@ FUNCTION(KOKKOS_ADD_EXECUTABLE_AND_TEST ROOT_NAME)
             )
         ENDIF()
     ENDIF()
-    # We noticed problems with -fvisibility=hidden for inline static variables
-    # if Kokkos was built as shared library.
-    IF(BUILD_SHARED_LIBS)
-      SET_PROPERTY(TARGET ${PACKAGE_NAME}_${ROOT_NAME} PROPERTY VISIBILITY_INLINES_HIDDEN ON)
-      SET_PROPERTY(TARGET ${PACKAGE_NAME}_${ROOT_NAME} PROPERTY CXX_VISIBILITY_PRESET hidden)
-    ENDIF()
 ENDFUNCTION()

 FUNCTION(KOKKOS_SET_EXE_PROPERTY ROOT_NAME)
```
4.4.0 is not more problematic than previous releases in the 4.X series (that is that bug being resolved in #7283 has always been there).  We just injected these visibility flags in the tests configuration.
